keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/DoubleVertexSamples.java

```diff
@@ -39,4 +39,7 @@ public DoubleTensor getVariances() {
             .divInPlace(samples.size()-1);
     }
 
+    public DoubleTensor asTensor() {
+        return DoubleTensor.stack(0, samples.toArray(new DoubleTensor[0]));
```

Comment:
`samples.stream().toArray(DoubleTensor[]::new)` is preferable since `[0]` is misleading